apps/transformers/test/unit/transformations/regex_replace_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule Transformers.RegexReplaceTest do
+  use ExUnit.Case
+
+  alias Transformers.RegexReplace
+
+  test "when source field not specified, return error" do
```

Comment:
Do you see any value in using a data test for these three (very similar) unit tests? Something like:

```
data_test "when #{missing_field} field not specified, return error" do
     payload = %{
       "something" => "123"
     }

     parameters = %{
       "regex" => "^(\\w+)",
       "replacement" => "a",
       "source" => "something"
     } |> Map.delete(missing_field)

     {:error, reason} = RegexReplace.transform(payload, parameters)

     assert reason == "Missing transformation parameter: #{error_message}"

     where([
        [:missing_field, :error_message],
        ["source", "sourceField"],
        ["regex", "regex"],
        ["replacement", "replacement"],
     ])
   end
```

Warning: The above is untested code.

Reply:
After some googling, I'm not finding documentation for data_test. Can you point me in the right direction?

Reply:
Nevermind, found it on the Chechov library. I'll try it out in this case and we can see how we like it.